src/client/components/Elevator/Elevator.component.js

```diff
@@ -0,0 +1,120 @@
+  },
+  {
+    header: 'EXAMPLES',
+    content:
+      '<p>I recently graduated from college with a degree in communications. I worked on the college newspaper as a reporter, and eventually, as the editor of the arts section. I&lsquo;m looking for a job that will put my skills as a journalist to work.</p><p>I create illustrations for websites and brands. My passion is coming up with creative ways to express a message, and drawing illustrations that people share on social media.</p>',
+  },
+  {
+    header: 'KEY TAKEAWAYS',
+    content:
+      '<h4>Keep it short and sweet:</h4><p>Your elevator speech is a sales pitch. Be sure you can deliver your message in 60 seconds or less.</p><h4>Focus on the essentials:</h4><p>Say who you are, what you do, and what you want to achieve.</p><h4>Be positive:</h4><p>Your time is limited. Focus on what you want to do, not what you do&lsquo;t want to do. Be upbeat and flexible.</p><h4>Practice, practice, practice:</h4><p>Deliver your speech to a friend or record it, so that you can be sure that your message is clear.</p>',
+  },
+];
+
+const ElevatorLevel = ({
+  header,
+  content,
+  floor,
+  onClickPrev,
+  onClickNext,
+}) => {
+  return (
+    <div className="elevator-level">
+      <h3 className="elevator-heading">{header}</h3>
+      <div
+        className="elevator-content"
+        dangerouslySetInnerHTML={{ __html: content }}
+      />
+      {floor > 0 && floor <= 5 ? (
+        <img
+          src={UpArrow}
+          alt="go to previous"
+          onClick={onClickPrev}
+          aria-hidden="true"
+          className="elevator-arrow elevator-up"
+        />
+      ) : (
+        <div />
+      )}
+      {floor >= 0 && floor < 5 ? (
+        <img
+          src={DownArrow}
+          alt="go to next"
+          onClick={onClickNext}
+          aria-hidden="true"
+          className="elevator-arrow elevator-down"
+        />
+      ) : (
+        <div />
+      )}
+    </div>
+  );
+};
+
+function Elevator({ level }) {
+  const [floor, setFloor] = useState(level);
+  const handleClickPrev = (e) => {
+    e.preventDefault();
+    setFloor((prev) => prev - 1);
+  };
+  const handleClickNext = (e) => {
+    e.preventDefault();
+    setFloor((prev) => prev + 1);
+  };
+  // will add this after pagination component is merged
+  // const handleClickPagination = (item) => {
+  //   setFloor(item);
+  // };
+  return (
+    <div className="elevator-component">
+      {/* will add this after pagination component is merged */}
+      {/* <Pagination floor={floor} handleClickPagination={handleClickPagination} /> */}
+      {elevatorSpeechArray
+        .filter((item, index) => {
+          return index === floor;
+        })
+        .map((item) => (
```

Comment:
Hi @TITHI1244 , In the map, you are missing key property...

Reply:
Oh thanks :) That I always do :D